src/encoded/schemas/cart.json

```diff
@@ -23,12 +23,19 @@
             "description": "The name of the cart",
             "type": "string"
         },
+        "identifier": {
+            "title": "URI-compatible cart identifier",
+            "description": "User-specified identifer for this cart that can be used in a URI for the cart.",
+            "comment": "This can only contain non-space characters, and must be unique througout the system.",
```

Comment:
small typo: "throughout" not "througout"

Reply:
Corrected